src/quiz/select.jsx

```diff
@@ -0,0 +1,51 @@
+/* This select component brings the option select  */
+const Option = () => {
+  const arraySelect = [
+    { value: "", option: "select an option" },
+    { value: "yes", option: "Yes" },
+    { value: "no", option: "No" },
+  ];
+
+  const optionMap = arraySelect.map((val) => {
```

Comment:
this is not a map but an array, we could call it optionsArray, or optionsComponents since you're mapping them to react components.